src/Conf.ml

```diff
@@ -1234,7 +1246,8 @@ let default_profile =
       C.default Formatting.space_around_collection_expressions
   ; type_decl= C.default Formatting.type_decl
   ; wrap_comments= C.default Formatting.wrap_comments
-  ; wrap_fun_args= C.default Formatting.wrap_fun_args }
+  ; wrap_fun_args= C.default Formatting.wrap_fun_args
+  ; end_of_line_in= C.default Formatting.end_of_line_in }
```

Comment:
same ordering comment as for the type definition

Reply:
Done